### PR TITLE
DIV-5062: Creating new fields for storing DA-related dates

### PR DESCRIPTION
**Pull request by jvictorcf**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-5062

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseField.json [hunk 1]
```diff
@@ -12346,5 +12346,61 @@
...
+    "CaseFieldID": "DateRespondentEligibleForDA",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "25/04/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DateRespondentEligibleForDA",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "R"
```
Comment (hosainnet) [inline definitions/divorce/json/AuthorisationCaseField.json]: Dunno if superuser usually have greater than R permission

Comment (qzhou-hmcts) [reply definitions/divorce/json/AuthorisationCaseField.json]: Yup, I'd advise CRUD for superuser in all the fields

**Comment by qzhou-hmcts** [comment]
Do you need to show the fields in any tab?

File: definitions/divorce/json/AuthorisationCaseField.json (hunk 1 above)
Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseField.json]: ok. I'll give it CRUD.
But shouldn't the superuser only be used for changing states?

**Comment by jvictorcf** [comment]
> Bump version number please, also I suppose there will be a CFS PR for this?

There will be a CFS PR for it, yes.

**Pull request merged**